storybook/pages/StatusChatInputPage.qml

```diff
@@ -0,0 +1,111 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Storybook 1.0
+import Models 1.0
+
+import utils 1.0
+import shared.status 1.0
+import shared.stores 1.0
+
+SplitView {
+    id: root
+
+    QtObject {
+        id: globalUtilsMock
```

Comment:
I'm thinking that maybe we can keep as a "rule" for the complete project to add all mocks into a separate and dedicated folder?? WDYT?

Reply:
We're going to work in this direction. There is the mocks folder and any reusable mocks could go there. Regarding Utils we can mock it and remove the duplicate code from almost all pages in a separate PR (I had a first attempt here #8606 and decided to split that PR)